x/consensus/keeper/keeper.go

```diff
@@ -23,8 +23,8 @@ type Keeper struct {
 	storeService storetypes.KVStoreService
 	event        event.Service
 
-	authority string
-	Params    collections.Item[cmtproto.ConsensusParams]
+	authority   string
+	ParamsStore collections.Item[cmtproto.ConsensusParams]
```

Comment:
We could avoid the conflict by creating a `msgServer` implementer wrapper, it's fine this way too.